docs/content/preview/manage/backup-restore/lightweight-db-clone.md

```diff
@@ -0,0 +1,261 @@
+```
+
+## How to use database cloning
+
+### Prerequisites
+
+- [Create a snapshot schedule](../../../manage/backup-restore/point-in-time-recovery/#create-a-schedule) for the database you want to clone.
+
+This keeps the history of the updates to a certain retention period. After that you can create a clone of the original database as of any point in time within the specified retention period.
+
+- You have to trust local YSQL connections (that use UNIX domain sockets) in the [host based authentication](../../../secure/authentication/host-based-authentication/). You have to do this for all the yb-tservers in the cluster. You can do this when starting the yb-tserver process by adding the authentication line `local all all trust` to the yb-tserver flag, [ysql_hba_conf_csv](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv).
+
+For example, if you are using yb-ctl you can run the following command:
+
+```sh
+./bin/yb-ctl create --master_flags "enable_db_clone=true,allowed_preview_flags_csv=enable_db_clone" --tserver_flags '"ysql_hba_conf_csv=""host all all 0.0.0.0/0 trust,local all all trust"""'
+```
+
+{{<note title="Note">}}
+Ensure to not override your default host based authentication rules when trusting the local connection. You may need to add additional authentication lines to `ysql_hba_conf_csv` based on your specific configuration. For more information, see [host based authentication](../../../secure/authentication/host-based-authentication/).
+{{</note>}}
+
+### Cloning in YSQL
+
+Because YugabyteDB is PostgreSQL compatible, you can create a database as a clone of another using the `TEMPLATE` SQL option of `CREATE DATABASE` command as follows:
+
+```sql
+CREATE DATABASE clone_db TEMPLATE original_db;
+```
+
+`clone_db` is created as a clone of database `original_db` which contains the latest schema and data of `original_db` as of current time.
+Additionally, to create a clone of the original database at a specific point in time in the past (within the history retention period specified when creating the snapshot schedule) you can specify the [Unix timestamp](https://www.unixtimestamp.com/) in microseconds in the `AS OF` option as follows:
+
+```sql
+CREATE DATABASE clone_db TEMPLATE origginal_db AS OF 1723146703674480;
+```
+
+### Cloning in YCQL
+
+You can create a clone in YCQL using the yb-admin command as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS clone_namespace ycql.originaldb1 clonedb2 1715275616599020
+```
+
+`clonedb2` is created as as a clone of the ycql database `originaldb1` as of 1715275616599020 Unix timestamp.
+
+### Check the clone status
+
+You can check the status of clone operations performed on a database with the `source_database_id` (YSQL)/`source_namespace_id`(YCQL) field using the yb-admin command `list_clones` as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS list_clones 00004000000030008000000000000000
```

Comment:
how do you find out what the db ID is so that you can use it in this command?

Reply:
cc @yamen-haddad